src/effects/backends/builtin/beatgrindeffect.cpp

```diff
@@ -53,22 +53,22 @@ EffectManifestPointer BeatGrindEffect::getManifest() {
     quantize->setRange(0, 1, 1);
 
     EffectManifestParameterPointer feedback = pManifest->addParameter();
-    feedback->setId("feedback_amount");
-    feedback->setName(QObject::tr("Feedback"));
-    feedback->setShortName(QObject::tr("Feedback"));
+    feedback->setId("amplitude");
+    feedback->setName(QObject::tr("Amplitude"));
+    feedback->setShortName(QObject::tr("Amplitude"));
     feedback->setDescription(QObject::tr(
-        "Amount the echo fades each time it loops"));
+        "Volume of loop"));
```

Comment:
Isn't this redundant with dry/wet?

Reply:
yes and no, you could, for example, go full wet with a low volume and increase the volume of the loop only

Reply:
That's what the chain's mix knob is for.

Reply:
Hmm, nice, I haven't found that knob yet, not sure how I missed it lol.
But I am still a confused with chains dry/wet modes.
Why do we need a separated dry/wet for this effect then?

Reply:
Every effect should have some way to set a knob so that it makes no change to the sound. I can't think of a way to do that for this effect besides a dry/wet knob.